components/toggleButton/toggleButton.js

```diff
@@ -15,27 +18,52 @@ export default function ToggleButton(props) {
     selectedIndex,
   } = props;
 
-  if (!Array.isArray(items) || items.length < 2) {
+  if (
+    (!Array.isArray(items) || items.length < 2) &&
```

Comment:
you should also hide the component if it receives only one child

Reply:
Done

Reply:
Why do we render `null` if we have only one item/child? I know it is an edge case and that most of the time it doesn't make sense to have a toggle button with only one option, but isn't it more predictable to just render a single item than nothing at all?

Reply:
I don't think so. It's a toggle, to render it you should attend some minimum requirements otherwise you can just render a Button.

Reply:
I understand, but I also believe that code should make the least assumptions possible, unless it can break the implementation. Someone may want to dynamically build a toggle button and still show the option even if it is a single one (instead of handling it as a special case to render something else), or they can check if they only have a single item and not render the toggle button in the first place.

Either way, this is probably out of the scope of this PR 🙂

Reply:
Well, if you want to use a Toggle Button, it means that you want to give more than one option to the user. If for some reason you only have one option (even with dynamic data) you should select this option and only show a Label or something indicating that the only option was selected. 

This kind of logic/validation should not be here.